src/mmw/apps/modeling/tasks.py

```diff
@@ -210,6 +241,19 @@ def _get_precip_value(model_input):
         return None
 
 
+def convert_result_areas(pixel_width, results):
+    """
+    Updates area values on the result set from tr55. TR55 gives area as a
+    number of pixels in a geotiff. This converts the area to square meters.
+    """
+    for i in range(len(results)):
+        categories = results[i]['categories']
+        for j in range(len(categories)):
+            print(categories[j])
```

Comment:
I assume this is here for debugging. Probably should be removed prior to merging.